src/sys/sys_main.c

```diff
@@ -913,17 +915,83 @@ void *Sys_LoadGameDll(const char *name, qboolean extract,
 	return libHandle;
 }
 
+void Sys_ParseArgsDrawBanner(FILE *stream)
+{
+	static int alreadyDrawn = qfalse;
+	if (!alreadyDrawn)
+	{
+		alreadyDrawn = qtrue;
+		fprintf(stream,
+		        "  ███████╗████████╗   ██╗     ███████╗ ██████╗  █████╗  ██████╗██╗   ██╗\n"
+		        "  ██╔════╝╚══██╔══╝██╗██║     ██╔════╝██╔════╝ ██╔══██╗██╔════╝╚██╗ ██╔╝\n"
+		        "  █████╗     ██║   ╚═╝██║     █████╗  ██║  ███╗███████║██║      ╚████╔╝\n"
+		        "  ██╔══╝     ██║   ██╗██║     ██╔══╝  ██║   ██║██╔══██║██║       ╚██╔╝\n"
+		        "  ███████╗   ██║   ╚═╝███████╗███████╗╚██████╔╝██║  ██║╚██████╗   ██║\n"
+		        "  ╚══════╝   ╚═╝      ╚══════╝╚══════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝   ╚═╝\n"
+		        "    Version: %s\n"
+		        "\n",
+		        PRODUCT_VERSION_STR
+		        );
+	}
+}
+
+void Sys_ParseArgsShowHelpAndExit(int ret)
+{
+	FILE *stream = (ret == 0) ? stdout : stderr;
+	Sys_ParseArgsDrawBanner(stream);
+	fprintf(stream,
+	        "Usage:\n"
+	        "  <etl-bin> [option]* -- [server options]*\n"
+	        "\n"
+	        "Options:\n"
+	        "  -h, --help                 Show help and exit\n"
+	        "  -v, --version              Show version and exit\n"
+	        "  -t, --title <title>        Set the game window title\n"
+	        "\n"
+	        "Server Options:\n"
```

Comment:
This isn't strictly `server options` as these can be used on the client too, more appropriate name would be `launch options` or `startup options`.